src/ResourceManager/Network/Commands.Network/help/Set-AzureRmPublicIpAddress.md

```diff
@@ -27,11 +27,11 @@ The **Set-AzureRmPublicIpAddress** cmdlet sets the goal state for a public IP ad
 ```
 PS C:\> $publicIp = Get-AzureRmPublicIpAddress -Name $publicIpName -ResourceGroupName $rgName
 
-PS C:\> $publicIp.PublicIpAllocationMethod = "Dynamic"
+PS C:\> $publicIp.PublicIpAllocationMethod = "Static"
```

Comment:
why do we need this change in the readme file?

Reply:
@chandrasekarsrinivasan this example description below says: Second command sets the allocation method of the public IP address object to "Static".